addons/website/static/src/scss/website.scss

```diff
@@ -200,6 +200,11 @@ font[class*='bg-'] {
         border-bottom-right-radius: $size;
     }
 }
+// Smaller container
+.o_container_small {
+    @extend .container;
+    max-width: $o-container-small-width;
+}
```

Comment:
In fact, using this feels strange because in mobile, container are bigger than the o_container_small, which causes a misalignment with the header container. In fact I would not define a variable at all: just use the bootstrap container max width of 'md' for >='lg' screens.
```scss
.o_container_small {
    @extend .container;
    @include media-breakpoint-up(lg) {
        max-width: map-get($container-max-widths, md);
    }
}
```

Something like that. What do you think?

I checked a lot of blogs and they all use ~720px wide text.

Reply:
620px was the width used in blog, conceptually I think your solution makes a lot more sense, but some blog layouts will be affected.

But indeed I should've only overridden the breakpoints that were larger than 620px instead of setting the max-width for all device sizes.

Reply:
I know, but 720px seems fine too, it's a bootstrap value and the value used on the 3 "popular" blogs I quickly checked. What do you think @sbuhl ?

Reply:
I didn't really understand, you want to change the width on the blog as well?
The width of the blog of bootstrap is 728: https://blog.getbootstrap.com/
On Medium, the width is 680. 
But I suppose 720 is fine.

Reply:
That moment bootstrap thinks their own lib is not good enough to use it for their blog :') Yes, obviously, technically the best thing to do is sharing that narrowed width of blogs with our new small container option. And technically, the best thing to do is either saying: the small container is max 540px width everywhere (which is a bit small in large screen to me) or max 540px in <=sm and max 720px otherwise, which seems nice to me. (especially since what I see on the web is around 720px yes, even bootstrap apparently) :+1: